src/Block/BlockContextManager.php

```diff
@@ -141,8 +141,8 @@ private function configureSettings(OptionsResolver $optionsResolver, BlockInterf
 
         // add type and class settings for block
         $class = ClassUtils::getClass($block);
-        $settingsByType = isset($this->settingsByType[$block->getType()]) ? $this->settingsByType[$block->getType()] : [];
```

Comment:
These kind of changes should be covered by their respective rule at `.php_cs.dist`. "ternary_to_null_coalescing" in this case.

Reply:
Agree, but if we add it, does this mean that flint is going to fail in every PR (that the repository has this kind of code)?